config/locales/fr.yml

```diff
@@ -52,7 +52,7 @@ account.email_language.name.fr: Français
 account.email_language.name.zh: 中文（简体）
 account.email_language.updated: Votre langue de préférence pour les e-mails a été mise à jour.
 account.emails.confirmed_html: Vous avez confirmé votre adresse e-mail. Rendez-vous sur <a href="%{url}">vos comptes connectés</a> pour actualiser l’adresse e-mail que vous communiquez aux organismes connectés.
-account.forget_all_browsers.longer_description: Une fois que vous aurez choisi d’« oublier tous les navigateurs », nous aurons besoin d’informations supplémentaires pour savoir que c’est bien vous qui vous connectez à votre compte. Nous vous demanderons une méthode d’authentification multi-facteurs (comme un code SMS/texto ou une clé de sécurité) chaque fois que vous souhaiterez accéder à votre compte.
+account.forget_all_browsers.longer_description: Une fois que vous aurez choisi d’«&nbsp;oublier tous les navigateurs&nbsp;», nous aurons besoin d’informations supplémentaires pour savoir que c’est bien vous qui vous connectez à votre compte. Nous vous demanderons une méthode d’authentification multi-facteurs (comme un code SMS/texto ou une clé de sécurité) chaque fois que vous souhaiterez accéder à votre compte.
```

Comment:
Do these need to be `_html` suffixed to avoid the `&nbsp;` showing as escaped HTML?

If so, I'd suggest one or both of the following:

- Making these changes in a separate pull request
- Using a unicode non-breaking space instead of the `&nbsp;` HTML entity, which shouldn't need the `_html` suffix since it's not HTML

For the latter, we've done this before, such as with the space in "state-issued ID" here:

https://github.com/18F/identity-idp/blob/ad5524df61a945c056db2c4b983e96dc7e59052e/config/locales/en.yml#L1909

It looks like a normal space (which can be confusing), but it's actual an irregular non-breaking space. VS Code shows these more clearly:

![image](https://github.com/user-attachments/assets/8641effc-5cbd-44e6-8628-1488227b31cd)

Reply:
yes they do need to be _html. but. yea I think a separate PR makes sense.